web/src/components/Inputs/DataStoreSelection/DataStoreSelection.tsx

```diff
@@ -18,57 +16,36 @@ interface IProps {
 const supportedDataStoreList = Object.values(SupportedDataStores);
 
 const DataStoreSelection = ({onChange = noop, value = SupportedDataStores.JAEGER}: IProps) => {
-  const {getFlag} = useCustomization();
-  const isLocalModeEnabled = getFlag(Flag.IsLocalModeEnabled);
   const {
     color: {text, primary},
   } = useTheme();
   const {dataStoreConfig} = useSettingsValues();
   const configuredDataStoreType = dataStoreConfig.defaultDataStore.type;
 
-  const handleChange = useCallback(
-    dataStore => {
-      const isDisabled = isLocalModeEnabled && dataStore !== SupportedDataStores.Agent;
-
-      if (!isDisabled) onChange(dataStore);
-    },
-    [isLocalModeEnabled, onChange]
-  );
-
   return (
-    <S.DataStoreListContainer tabPosition="left" onChange={handleChange}>
+    <S.DataStoreListContainer tabPosition="left" onChange={dataStore => onChange(dataStore as SupportedDataStores)}>
       {supportedDataStoreList.map(dataStore => {
-        if (dataStore === SupportedDataStores.Agent && !getFlag(Flag.IsAgentDataStoreEnabled)) {
-          return null;
-        }
-
         const isSelected = value === dataStore;
         const isConfigured = configuredDataStoreType === dataStore && dataStoreConfig.mode === ConfigMode.READY;
-        const isDisabled = isLocalModeEnabled && dataStore !== SupportedDataStores.Agent;
```

Comment:
are we removing the disabled datastores for localMode? how is this going to look for localMode?